models/bases/encounters.sql

```diff
@@ -1,15 +1,13 @@
 select
     id,
     start_date::timestamp as start_datetime,
-    case when end_date < start_date then start_date::timestamp
-        else end_date::timestamp
-    end as end_datetime,
+    case when end_date < start_date then start_date::timestamp else end_date::timestamp end as end_datetime,
```

Comment:
I know we've removed line lengths before wrap around but I still think for readability it would be nice to keep each case statement on a separate line.
```suggestion
    case when end_date < start_date then start_date::timestamp
        else end_date::timestamp
    end as end_datetime,
```